src/main/scala/cromwell/binding/command/Command.scala

```diff
@@ -49,7 +50,15 @@ case class Command(parts: Seq[CommandPart]) {
    * @param parameters Parameter values
    * @return String instantiation of the command
    */
-  def instantiate(parameters: Map[String, WdlValue]): String = normalize(parts.map { part => part.instantiate(parameters) }.mkString(""))
+  def instantiate(parameters: Map[String, WdlValue]): Try[String] = {
+    // http://stackoverflow.com/questions/15495678/flatten-scala-try
```

Comment:
My understanding here is that because multiple `instantiate()` calls may return a Try, we need to translate a `Seq[Try[String]]` to a `Try[Seq[String]]` so we can do the map and normalize?  I still am not 100% clear on this... my hope is that we could make this simpler somehow...